README.md

```diff
@@ -57,6 +57,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
   - `<textarea>`
   - `<video controls>`
 - Checks the `dir` attribute is only set to `ltr`, `rtl` or `auto`
+- Checks the `video` attribute to ensure `autoplay` is not set
```

Comment:
Nice work! Could we change one thing? Could we remove the `video` check? We'd like to check for whether `autoplay` exists anywhere, such as `video` and `audio`. So I think the below will be ok:

```css
[autoplay] {
  ...
}
```

Reply:
Ok will change it now!